scripts/release_info.py

```diff
@@ -292,6 +310,188 @@ def _check_storage_models(current_release):
     return [item for item in diff_list if item.startswith('core/storage')]
 
 
+def get_blocking_bug_issue_count(repo):
+    """Returns the number of unresolved blocking bugs.
+
+    Args:
+        repo: github.Repository.Repository. The PyGithub object for the repo.
+
+    Returns:
+        int: Number of unresolved blocking bugs.
+    """
+    blocking_bugs_milestone = repo.get_milestone(number=39)
+    return blocking_bugs_milestone.open_issues
+
+
+def check_prs_for_current_release_are_released(repo):
+    """Checks that all pull requests for current release have a
+    'PR: released' label.
+
+    Args:
+        repo: github.Repository.Repository. The PyGithub object for the repo.
+
+    Returns:
+        bool. Whether all pull requests for current release have a
+            PR: released label.
+    """
+    all_prs = repo.get_pulls(state='all')
+    for pr in all_prs:
+        label_names = [label.name for label in pr.labels]
+        if 'PR: released' not in label_names and (
+                'PR: for current release' in label_names):
+            return False
+    return True
+
+
+def update_sorted_file(filepath, new_list, start_index_line):
```

Comment:
Actually, I just realized that a bunch of these functions are in the wrong place. release_info.py is a script that generates a doc, release_info.md. It should not be changing files or have other side effects -- that should happen as part of the release process in a separate script that actually does the release process.

I think people expect release_info to be like a GET action, i.e. it generates an info doc outside the oppia folder and doesn't modify state elsewhere. We should preserve that.

Reply:
Updated.